tests/integration/test_properties.py

```diff
@@ -152,8 +152,10 @@ def test_archive_property_by_id(self, valid_header, create_property):
         test_property = create_property()
 
         """The archive property endpoint should return a 200 code when successful"""
-        responseSuccess = self.client.post(
-            f"/api/properties/archive/{test_property.id}", headers=valid_header
+        responseSuccess = self.client.put(
+            f"/api/properties/{test_property.id}",
+            headers=valid_header,
+            json={"archived": not test_property.archived},
```

Comment:
When we create a property the archive value is set to False unless otherwise specified. So the value being passed into archived would be True. I do find this line a bit confusing to read, and I can see how it would be more confusing if someone doesn't have this knowledge.
Perhaps we can change `test_property = create_property()` to `test_property = create_property(archived=False)`
And line 158 to:
```suggestion
            json={"archived": True},
```

Reply:
Your suggestion makes sense. The intention is to test that a property is properly archived. Using ```json={"archived": True}``` makes the intention clear.